src/components/NoteExpand/CommentsBox/CommentsList/CreateComment/CreateComment.tsx

```diff
@@ -0,0 +1,60 @@
+import SendIcon from '@mui/icons-material/Send';
+import Button from '@mui/material/Button';
+import { useFormik } from 'formik';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+
+import { addComment } from '../../../../../redux/notes-slice';
+import styles from './CreateComment.module.scss';
+
+const CreateComment = () => {
+  const dispatch = useDispatch();
+  const { id } = useParams<{ id: string }>() || undefined;
+  const initialFormValues = {
+    id: `${id}`,
```

Comment:
I guess it should be `noteId: id,`

Reply:
This is comments, maybe `commentId: id,` ?

Reply:
sure, `commentId`

Reply:
complete